tests/test_sanity.py

```diff
@@ -19,3 +22,14 @@ def test_docker_deamon_is_running():
     unit = Unit(b'docker.service')
     unit.load()
     assert unit.Unit.ActiveState == b'active', "Docker deamon is not running"
+
+
+def test_docker_container_is_running():
+    """
+    Check that a given container name is running.
+    client.containers.list() is similar to 'docker ps'. it returns only running containers by default.
+    """
+    client = docker.from_env()
+    assert WEB_APP_CONTAINER_NAME in [con.name for con in client.containers.list()], (
+        f"Container {WEB_APP_CONTAINER_NAME} is not running."
+    )
```

Comment:
I think that inspec is a better suited technology for testing infrastructure.

Reply:
Currently there is no inspec in our code, and i wanted to make a test for running container.
python 'docker' library is a convenient way of using docker API.

Reply:
Please elaborate why it is convenient. Stating that one approach is more convenient than another is not something I or anyone else for that matter can learn from.
You can also defer the migration to inspec in a future task. I would accept the current solution as a "quick and dirty" one.

Reply:
Alright, lets consider this as a temporary solution.
Integrating rspec will require some effort, and learning curve. While doing that, this patch will be thrown far back to the backlog.

Reply:
I am not following. We have decided that the pytest approach is accepted for now. The inspec will be implemented right after this one is merged.

Reply:
Ok, thats what i was saying. Thats totally fine with me.
You were saying that inspec is a better suited here. I believe this is true.
But for sake of this patch, there is no argue IMHO.

Reply:
Please file an issue for the inspec effort. Start working on it after this PR is merged.

Reply:
Created issue:
https://github.com/beyond-io/beyond/issues/190